src/Common/Commands.Common/AzurePSCmdlet.cs

```diff
@@ -671,7 +671,7 @@ protected override void ProcessRecord()
                 base.ProcessRecord();
                 ExecuteCmdlet();
             }
-            catch (Exception ex)
+            catch (Exception ex) when (!(ex is PipelineStoppedException) && ex.InnerException != null)
```

Comment:
Let's make this a template method, called IsTerminatingError - make this a virtual method in AzureRM so that cmdlets can override it to add their own terminating exceptions.